pkg/endpoint/endpoint.go

```diff
@@ -82,6 +83,12 @@ var (
 		Requires:    []string{OptionConntrack},
 	}
 
+	OptionSpecConntrackLocal = option.Option{
+		Define:      "CONNTRACK_LOCAL",
+		Description: "Use container local connection tracking table instead of global one",
```

Comment:
@tgraf oh I was asking the `local` vs `dedicated`. The word container was already there.